relay-event-normalization/src/normalize/span/description/redis.rs

```diff
@@ -0,0 +1,591 @@
+use std::cmp::Ordering;
+
+/// Returns a redis command if it is a case-insensitive prefix of `seek`.
+pub fn matching_redis_command(seek: &str) -> Option<&str> {
+    let seek = seek.as_bytes();
+    let index = REDIS_COMMANDS
+        .binary_search_by(|probe| {
+            let probe = probe.as_bytes();
+            // step 1 - compare prefixes
```

Comment:
It would be great if you can add a few more comments here and some basic examples for the branches/match arms, this is quite hard to think through without.

Same for the second check.